packages/edit-site/src/components/save-panel/index.js

```diff
@@ -150,8 +150,7 @@ export default function SavePanel() {
 				} ) }
 			>
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This can be seen, when you open the save panel. It has style for `screen-reader-text`, hence need to remove that style to see the visual change.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="open-save-panel-before" src="https://github.com/user-attachments/assets/9c74ac5e-24d8-45f4-ba04-042fab06d0e8"> | <img width="419" alt="Open-save-panel-after" src="https://github.com/user-attachments/assets/1f438cd9-415f-4ef9-a1b7-1044d25c8c78"> |